R/utils.R

```diff
@@ -267,3 +262,50 @@ sum_vectors_in_list <- function(lst) {
   )
   .rowSums(tmp, m = len1, n = length(lst))
 }
+
+#' Confidence interval calculations which are common across effect results.
+#' @keywords internal
+h_confint <- function(x, parm, level = 0.95, transform, include_se = FALSE, ...) {
+  assert_matrix(x)
+  assert_names(colnames(x), must.include = c("Estimate", "Std.Err"))
+  assert_names(rownames(x), type = "unique")
+  assert_number(level, lower = 0, upper = 1)
+  assert_flag(include_se)
+  if (!missing(parm)) {
+    assert(
+      check_integerish(parm, lower = 1, upper = nrow(x)),
+      check_subset(parm, row.names(x))
+    )
+  }
+  if (!missing(transform)) {
+    assert_function(transform)
+  }
+  est <- x[, "Estimate"]
+  se <- x[, "Std.Err"]
+  z <- qnorm((1 + level) / 2)
+  ret <- matrix(
+    c(
+      est,
+      if (include_se) se else NULL,
```

Comment:
if transformed the se does not make any sense anymore

Reply:
sure. This is a helper function only and not part of the user interface, so it is up to the calling functions to use the `include_se` argument as it makes sense.